### PR TITLE
Setup PostgreSQL database service

### PR DESCRIPTION
**Pull request by libdx**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Dockerfile
+Dockerfile-prod
+.dockerignore
+.git
+.gitignore
```
Comment (pioole) [inline .dockerignore]: Add *.pyc files

File: docker-compose-prod.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+services:
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: gunicorn -b 0.0.0.0:5000 myflights.wsgi
```
Comment (pioole) [inline docker-compose-prod.yml]: super. ;)

File: .dockerignore (hunk 1 above)
Comment (libdx) [reply .dockerignore]: Done

Comment (pioole) [reply .dockerignore]: Oh, sorry, my bad. I read this as .gitignore...
No need for .pyc here.

**Pull request merged**